src/ml/neural_net/mps_compute_context.mm

```diff
@@ -25,7 +25,8 @@
 // At static-init time, register create_mps_compute_context().
 // TODO: Codify priority levels?
 static auto* mps_registration = new compute_context::registration(
-    /* priority */ 0, &create_mps_compute_context);
+    /* priority */ 1, &create_mps_compute_context);
```

Comment:
Shreya is working on solving this problem as we speak... Her changes should make it so you don't have to do this, FYI